Client/Frontend/Browser/WebView/WebviewViewController.swift

```diff
@@ -35,9 +38,18 @@ class WebviewViewController: UIViewController, ContentContainable, Screenshotabl
         ])
     }
 
+    private func setupScreenTimeController() {
+        addChild(screenTimeController)
+        screenTimeController.view.frame = webView.frame
```

Comment:
I think we should use constraints here as well instead of setting a frame to avoid problems with rotation or other screen size changes.